frontend/src/views/CellCards/common/store/index.ts

```diff
@@ -0,0 +1,10 @@
+import { createContext, Dispatch } from "react";
```

Comment:
@ashin-czi Let's remove the entire `store/` folder. We don't have any reducers yet.